bosch_thermostat_client/gateway/base.py

```diff
@@ -64,18 +64,18 @@ def __init__(self, host):
         self.initialization_msg = None
         self._bus_type = None
 
-    def get_base_db(self):
-        return get_initial_db(self.device_type)
+    async def get_base_db(self):
```

Comment:
This change..

Reply:
This is not a breaking change as it is only used in `BaseGateway.initialize()` which is already `async`.